classical/python-sdk/data-science/src/evaluate.py

```diff
@@ -40,10 +43,10 @@ def parse_args():
 def main():
     # Parse command-line arguments
     args = parse_args()
-    prepared_data_path = os.path.join(args.prepared_data_path, run.parent.id)
-    model_path = os.path.join(args.model_path, run.parent.id)
-    explainer_path = os.path.join(args.explainer_path, run.parent.id)
-    evaluation_path = os.path.join(args.evaluation_path, run.parent.id)
+    prepared_data_path = args.prepared_data_path
```

Comment:
@chrey-gh This has been updated since you last did a git pull I think. The "run.parent_id" is the newest feature.

Reply:
as with the subsequent lines too